packages/styles/src/mixins/_utilities.scss

```diff
@@ -8,10 +8,10 @@
 @use '../variables/commons';
 @use '../variables/breakpoints';
 
+@use './list' as list-mx;
+
 @mixin reset-list() {
-  margin-block: 0;
-  padding-inline-start: 0;
-  list-style: none;
+  @include list-mx.list-unstyled();
```

Comment:
reset-list is being used on the stepper and the layout seems broken now.
I'm not sure reset-list() was doing the same as list-unstyled.

Reply:
I've tryed to fix the stepper component, but after some attempts, I resetted the reset-list mixin to its original state, so we don't mess up anything already using it.